types/spectest/generate/tests.json

```diff
@@ -1 +1 @@
N0tSL0hPSGgKNHhudmVjZU5zQkw0TTdnMmZuT3QxbmdNY1FGbEs3cjBncUZWSk1DVDVmMnVlRWxFTTNvbS9Mc0Z0bGtnRlZNVgpjeFhUOE13M2pnbzJmcEYzN0Vld1N4bkovNVdsbms5TmdueVhYSmpXR0ZmT3BPVWQ2bTNQS2RjTmR5bGNOeVpICnlRSURBUUFCCi0tLS0tRU5EIFJTQSBQVUJMSUMgS0VZLS0tLS0K","PlainText":"aGVsbG8gd29ybGQ=","CipherText":"FYS8gmCj3MAFbdddj9+v58igmgwfHLAxyY01sIovp96Eb99EIqxvmQBlXfvyDjBJVDeGoHsqEmQpBcMhWI7Kgg1pjNySJLfrE4Sbe36pMQ15iArlYT8tsL4UihzrDn3npZx8UmFnf+plcIkFPNULQ/pvLZNyBjvk278dN+7IDPi9oesFfAw+PyO/vq4KqEvqYmVw1dfa3gkWzh/BavNs6QgtdJyBGKQ893xgeNSpz8NP/Mpowuw0csbHbnjPSBQ3QhE48UKtVhHdlAP647Y4flKVYfXQPivSi6X6xEr2EPOSuPXLmCMMmC7rky1dyUa3sFpuJMEfHZj7IFDryGtSjg=="},"*partialsigmessage.EncodingTest_SignedPartialSignatureMessage encoding":{"Name":"SignedPartialSignatureMessage encoding","Data":"bAAAALgezSLkqlWt6mqT31w6CdHAWSVeTXP6ItK6HPuQMykaUpHcVx/4Oclv8eX49I/AVwBFjTiPPidWjw12BNR26jsY5Sb5aLFqaScq0u4m/lujP8QFOYoqJiUi1mOJHjO1IAEAAAAAAAAAAgAAAAAAAAAMAAAAAAAAABQAAACkebAj2fKOuczdNbuWD7XCZVMN/xeK5+Spt38Bf3WcpaISXnJr3gzZKjMwCVSUyMAHgqJ/qJB+jBqt49rNuOBhRobGfh9igwRgXXhh4O2YA8auIWtKrlWo/7Se7Rk8vqdGJhHSxZK99SFPEk/npSaatz1E78nmzhq+MvgZauUm3gEAAAAAAAAA","ExpectedRoot":[135,210,231,183,128,210,66,213,218,41,56,104,5,42,120,80,50,237,24,60,163,179,110,246,242,146,222,252,161,20,160,115]},"*share.EncodingTest_share encoding":{"Name":"share encoding","Data":"AQAAAAAAAACOgAZlUagbMYJYcJ7a990fY81oag5NuLKbu3rP5lYIZ3r1pSfZRI7keDVIXgK1C8CX2UqBHWRJNgUqnS1KzWRxiXTPZ42+/SZzJMjp+OcDSkprfYd54noBlTjDxCXD5eekAAAAAAADAVNZU7WmBAB0lIzxheqn0qu9ZoCPAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAAAAAAAAAJfZSoEdZEk2BSqdLUrNZHGJdM9njb79JnMkyOn45wNKSmt9h3niegGVOMPEJcPl5wIAAAAAAAAAprzr4wl9dBcbQMcSoDHOsDcds9PEAs8s5pG5Eg87q3XU1W36DzdZFUSZm/GMU1PtAwAAAAAAAACAkOC3ZmpF7MXU73QYrJnwnmyyRAKf6moKf8y+nuCLxTneDfpmvmuryEk92Z3RqY0EAAAAAAAAAKfAona3Cl7jOVx9bSZV7WBSiqJS1NIeyl/MUiPlDMQr0WiygKt6tTFzVN3fzz355A==","ExpectedRoot":[179,115,251,110,55,46,177,200,236,20,174,62,142,228,236,192,195,119,100,208,16,212,156,138,255,44,110,223,13,58,114,109]},"*ssvmsg.EncodingTest_SSVMessage encoding":{"Name":"SSVMessage encoding","Data":"AQAAAAAAAAAAAAMBjoAGZVGoGzGCWHCe2vfdH2PNaGoOTbiym7t6z+ZWCGd69aUn2USO5Hg1SF4CtQvABAAAAEQAAABsAAAAiuAHYeBnoJLPQ4DeKS6iGxaGwMQf3qYwHpyTLnbWI0Putu/2THkdv+/B0mllAGV9COw3lOeAOUA63gyjwQ7X4gBQqDkNccxVyQhtUHTCQOVZj5O7CDrcx0wMxnTp7LnRAQAAAAAAAAAAAAAAAAAAAAwAAAAAAAAAFAAAAIHOTUAW4aKab+DnYipc6IDHDjlZiSGQO5jekCbedMinIW27NQoWqCvP4M1Q0QdSMhaPMr2BH583R8fIBuFRHSRGXrbVQGRMJbhyxw051pAbDo9t6ItlkNEK6VMCHFYoa59BVr5kSYEeTm38fcbD8K3As9jUyxyxqhWSVjHvd+pZAQAAAAAAAACFI30D183QisXzZKefFPWC1wKdLVHdJ8fsYbfIzNTwWMPXVAxBDeShVYGPyv4yneALhKvGPWXFskxyBk6jaDFK99jD5tUW6OCYArESwcL82Cz2OpV3yOZTdKoz+RLRKpr0gCLeQ0LyvqIfzhBbri51Y/fx8Rtwn7Cg/78lKpY5DAEAAAAAAAAAgeoGNlGGzJ2/2j3YX1mLxd7YTzHU/2WIEPnvvwKTf4sYe8gG3KqJT8r+3Uph/mKVANj69KcWGetrcmwC3gZmDnC8JMcXhNKHsoao1LlNW1rzmU9b3FOWkVb0VJQXRithpzXy6V4FAJC0PPbGfBBkUQTq9CKF+UQ+1bmWHSCOBB8BAAAAAAAAAA==","ExpectedRoot":[35,127,95,31,85,239,156,36,192,176,9,106,147,66,178,138,51,163,36,229,98,241,185,201,55,2,156,124,103,186,68,21]}}
```

Comment:
@olegshmuelov, @GalRogozinski can someone explain me why is this changed?

Reply:
removed